manifests/init.pp

```diff
@@ -328,75 +148,6 @@
-    ssl_cert              => $::certs::foreman_proxy::proxy_cert,
-    ssl_key               => $::certs::foreman_proxy::proxy_key,
-    ssl_ca                => $::certs::foreman_proxy::proxy_ca_cert,
-    foreman_ssl_cert      => $::certs::foreman_proxy::foreman_ssl_cert,
-    foreman_ssl_key       => $::certs::foreman_proxy::foreman_ssl_key,
-    foreman_ssl_ca        => $::certs::foreman_proxy::foreman_ssl_ca_cert,
-    tftp                  => $tftp,
-    tftp_syslinux_root    => $tftp_syslinux_root,
-    tftp_syslinux_files   => $tftp_syslinux_files,
-    tftp_root             => $tftp_root,
-    tftp_dirs             => $tftp_dirs,
-    tftp_servername       => $tftp_servername,
-    bmc                   => $bmc,
-    bmc_default_provider  => $bmc_default_provider,
-    dhcp                  => $dhcp,
-    dhcp_listen_on        => $dhcp_listen_on,
-    dhcp_option_domain    => $dhcp_option_domain,
-    dhcp_interface        => $dhcp_interface,
-    dhcp_gateway          => $dhcp_gateway,
-    dhcp_range            => $dhcp_range,
-    dhcp_nameservers      => $dhcp_nameservers,
-    dhcp_vendor           => $dhcp_vendor,
-    dhcp_config           => $dhcp_config,
-    dhcp_leases           => $dhcp_leases,
-    dhcp_key_name         => $dhcp_key_name,
-    dhcp_key_secret       => $dhcp_key_secret,
-    dns                   => $dns,
-    dns_managed           => $dns_managed,
-    dns_provider          => $dns_provider,
-    dns_zone              => $dns_zone,
-    dns_reverse           => $dns_reverse,
-    dns_interface         => $dns_interface,
-    dns_server            => $dns_server,
-    dns_ttl               => $dns_ttl,
-    dns_tsig_keytab       => $dns_tsig_keytab,
-    dns_tsig_principal    => $dns_tsig_principal,
-    dns_forwarders        => $dns_forwarders,
-    virsh_network         => $virsh_network,
-    realm                 => $realm,
-    realm_provider        => $realm_provider,
-    realm_keytab          => $realm_keytab,
-    realm_principal       => $realm_principal,
-    freeipa_remove_dns    => $freeipa_remove_dns,
-    register_in_foreman   => $register_in_foreman,
-    foreman_base_url      => $foreman_url,
-    trusted_hosts         => [$parent_fqdn, $capsule_fqdn],
-    registered_proxy_url  => "https://${capsule_fqdn}:${capsule::foreman_proxy_port}",
-    oauth_effective_user  => $foreman_oauth_effective_user,
-    oauth_consumer_key    => $foreman_oauth_key,
-    oauth_consumer_secret => $foreman_oauth_secret,
-    templates             => $templates,
-  }
-
-  if $remote_execution_ssh {
-    class { '::foreman_proxy::plugin::remote_execution::ssh':
-        enabled           => $remote_execution_ssh,
-        generate_keys     => $remote_execution_ssh_generate_keys,
-        ssh_identity_dir  => $remote_execution_ssh_identity_dir,
-        ssh_identity_file => $remote_execution_ssh_identity_file,
-        require           => Class['::foreman_proxy'],
-    }
-  }
-
   if $pulp {
```

Comment:
@ehelms, Since the capsule-installer today provides the option to enabled/disable pulp, I would assume that we should maintain that for a capsule scenario.